plugin/pkg/scheduler/scheduler.go

```diff
@@ -185,7 +185,7 @@ func (sched *Scheduler) schedule(pod *v1.Pod) (string, error) {
 }
 
 // assume signals to the cache that a pod is already in the cache, so that binding can be asnychronous.
-func (sched *Scheduler) assume(pod *v1.Pod, host string) error {
+func (sched *Scheduler) assume(pod *v1.Pod, host string) (*v1.Pod, error) {
```

Comment:
wow, yes. We did not use object from cache directly; prefer to highlight copy outside `assume`:

```
assumedPod := *pod

sched.assume(assumedPod)
go sched.bind(assumedPod)
```